lib/puppetx/simp/compliance_mapper.rb

```diff
@@ -138,9 +138,8 @@ def initialize(object)
     def load(options={}, &block)
       @callback.debug("callback = #{callback.codebase}")
 
-      module_scope_compliance_map = callback.call_function('lookup', 'compliance_markup::compliance_map', { 'default_value' => {} })
-      top_scope_compliance_map    = callback.call_function('lookup', 'compliance_map', { 'default_value' => {} })
-
+      module_scope_compliance_map = block.call 'compliance_markup::compliance_map', {}
```

Comment:
This seems to work, but do we know where this `block` is coming from originally? Also, this doesn't have a `default_value`, does this work if nothing is found?